tools/src/main/java/org/apache/kafka/tools/AclCommand.java

```diff
@@ -249,8 +247,8 @@ private static Map<ResourcePatternFilter, Set<AccessControlEntry>> getProducerRe
         Set<ResourcePatternFilter> transactionalIds = filters.stream().filter(f -> f.resourceType() == ResourceType.TRANSACTIONAL_ID).collect(Collectors.toSet());
         boolean enableIdempotence = opts.options.has(opts.idempotentOpt);
 
-        Set<AccessControlEntry> topicAcls = getAcl(opts, new HashSet<>(Arrays.asList(WRITE, DESCRIBE, CREATE)));
-        Set<AccessControlEntry> transactionalIdAcls = getAcl(opts, new HashSet<>(Arrays.asList(WRITE, DESCRIBE)));
+        Set<AccessControlEntry> topicAcls = getAcl(opts, new HashSet<>(List.of(WRITE, DESCRIBE, CREATE)));
+        Set<AccessControlEntry> transactionalIdAcls = getAcl(opts, new HashSet<>(List.of(WRITE, DESCRIBE)));
```

Comment:
```suggestion
        Set<AccessControlEntry> topicAcls = getAcl(opts, Set.of(WRITE, DESCRIBE, CREATE));
        Set<AccessControlEntry> transactionalIdAcls = getAcl(opts, Set.of(WRITE, DESCRIBE));
```

Reply:
The original code used `new HashSet<>(List.of(...))`, so I believe the order was not guaranteed in the previous approach.